dll/win32/kernel32/CMakeLists.txt

```diff
@@ -49,7 +49,7 @@ list(APPEND SOURCE
     client/console/history.c
     client/console/init.c
     client/console/readwrite.c
-    client/console/vista.c
+    #client/console/vista.c
```

Comment:
Add a comment saying this file is included in kernel32_vista?

Reply:
I'll remove this entirely.